src/store/thunks/posts.js

```diff
@@ -14,6 +17,9 @@ export const fetchPosts = ({ sortBy, time }) => (dispatch) => {
           const postObj = post.data();
           postObj.id = post.id;
           postArray.push(postObj);
+          postArray.sort(function (a, b) {
```

Comment:
So the logic is that whenever we fetch posts of any kind we want the newest to oldest?

Reply:
Yes, this function sorts the array based on a value inside the array
I use it for newest to oldest, and for most popular